README.md

```diff
@@ -63,7 +63,10 @@ Clone the project locally. Note the submodule link to the original project(s).
 git clone https://github.com/Start9Labs/hello-world-wrapper.git
 cd hello-world-wrapper
 ```
-
+During the 0.3.0 beta (you won't need this after merge to main), run:  
+```
+git pull --rebase origin integration/0.3.0
```

Comment:
In this instance, after having cloned, we just need to `git checkout integration/0.3.0`

Other than that small change, this looks good